src/main/java/it/gov/pagopa/debtposition/controller/pd/crud/api/v1/IDebtPositionController.java

```diff
@@ -657,14 +655,15 @@ ResponseEntity<PaymentPositionModelBaseResponse> getDebtPositionByIUV(
       })
   @PatchMapping(
       value = "/organizations/{organizationfiscalcode}/transfers",
-      produces = {MediaType.TEXT_PLAIN_VALUE},
+      produces = {MediaType.APPLICATION_JSON_VALUE},
       consumes = {MediaType.APPLICATION_JSON_VALUE})
-  ResponseEntity<String> updateTransferIbanMassive(
+  ResponseEntity<UpdateTransferIbanMassiveResponse> updateTransferIbanMassive(
       @Parameter(
               description = "Organization fiscal code, the fiscal code of the Organization.",
               required = true)
           @PathVariable("organizationfiscalcode")
           String organizationFiscalCode,
       @Parameter(description = "The old iban to replace") @RequestParam @NotBlank String oldIban,
+      @Parameter(description = "Number of Transfer to update, max & default 1000") @Max(1000) @RequestParam(required = false, defaultValue = "1000") int limit,
```

Comment:
I prefer duplication in description, is more explicit:

_Number of Transfer to update (max = 1000, default = 1000)_

agree?